snowflake/ingest/utils/network.py

```diff
@@ -1,11 +1,5 @@
-# import to use ocsp module in python connector
-# this import will inject ocsp check method into
-# botocore.vendored.requests library
-import snowflake.connector
```

Comment:
Let's still leave this import since this will inject request library with OCSP.

Reply:
Added back!